app/services/nomis/feature.rb

```diff
@@ -0,0 +1,28 @@
+module Nomis
+  class Feature
```

Comment:
I prefer to be explicit of where business domain classes and objects are being called, worth the typing!